cartridges/app_custom_page_designer/cartridge/controllers/Home.js

```diff
@@ -0,0 +1,28 @@
+"use strict";
+
+var server = require("server");
+server.extend(module.superModule);
+
+var pageMetaData = require('*/cartridge/scripts/middleware/pageMetaData');
+
+server.append("Show", function (req, res, next) {
+        var Site = require("dw/system/Site");
+        var PageMgr = require("dw/experience/PageMgr");
+        var pageMetaHelper = require("*/cartridge/scripts/helpers/pageMetaHelper");
+
+        pageMetaHelper.setPageMetaTags(req.pageMetaData, Site.current);
+
+        var page = PageMgr.getPage("christmas-homepage");
```

Comment:
The requirement says this to be coming from site pref. Also please avoid having hard-coded values within the code.